app/settings/surveys/attribute-editor.html

```diff
@@ -106,8 +109,10 @@
         </div>
    </div>
    <div class="form-field">
```

Comment:
I already added an warning for duplicate messages which looks something like this:
![image](https://user-images.githubusercontent.com/70752431/95746896-2720b180-0cb5-11eb-8268-d2c98eb4b712.png)

The code for same lies at L30 of the same file.
Do you mean that I should add it below the form instead of adding it above them?